### DISCUSSION TITLE
Authentication unsuccessful for Outlook

### DISCUSSION THREAD

**Discussion by Rory11** [emailjs-com/help#28] [Q&A]
I am trying to set up a contact form and when setting up the service I received and Outlook invalid login: Authentication Unsuccessful, account locked error. I retried by leaving the test email unchecked as I saw in one of the other discussions and it worked. However when testing the form from the website I am getting that same error in the console and the form is not sending. Does anyone know what is causing this and how to work around it?

**Comment by xr0master** [comment]
"I retried by leaving the test email unchecked as I saw in one of the other discussions and it worked."
This does not solve any problems, it just keeps the service as is without testing. In some cases, for transactional services, this is relevant due to the inaccurate sender address.

If you are using Outlook 1st version, you have to create an application password. For Outlook v2 there shouldn't be any such problems.

**Reply by Rory11** [reply]
I was using version one and created the password, I didn't see anywhere where I should have added it inside the code if that's what you mean but perhaps I missed it. I was able to create a service using a gmail email service and then adding the outlook email into the template. Not sure how that affects the email visually on outlook as I don't use it but it but the client forward the test to me to let me know it was working and I didn't hear any complaints so I assume it's about the same. Thank you for your response though, I was trying to avoid having to ask for the password to their email account if possible but I may need to.
